doc/developer-notes.md

```diff
@@ -423,27 +423,38 @@ and its `cs_KeyStore` lock for example).
 Threads
 -------
 
-- ThreadScriptCheck : Verifies block scripts.
+- Main thread : Started from main() in bitcoind.cpp. Responsible for starting up
+  and shutting down the application.
 
-- ThreadImport : Loads blocks from `blk*.dat` files or `-loadblock=<file>`.
+- ThreadImport : Loads blocks from `blk*.dat` files or `-loadblock=<file>` on startup.
 
-- ThreadDNSAddressSeed : Loads addresses of peers from the DNS.
+- ThreadScriptCheck : Parallel script validation threads for transactions in blocks.
```

Comment:
```suggestion
- [`ThreadScriptCheck`](https://doxygen.bitcoincore.org/validation_8cpp.html#a925a33e7952a157922b0bbb8dab29a20) : Parallel script validation threads for transactions in blocks.
```

What about adding links to the thread, so that when it is removed in the future, it will be immediately clear from the dead link?